contrib/triton-aml/README.md

```diff
@@ -0,0 +1,36 @@
+Triton-AML
+======
+
+*Triton-AML* is a Triton custom backend running with Marian in the AzureML Inference Environment, it's one of the implementation of [Triton Backend Shared Library](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend.html#backend-shared-library).
+
+This backend is compiled with the static library of Marian on a specific version.
+
+Layout:
+
+- marian_backend: Triton Marian backend source code
+- src: Changed code and CMakeLists.txt of Marian
+- Dockerfile: Used for compiling the backend with the static library of Marian
+- build.sh: A simple shell script to run the Dockerfile to get the generated libtriton_marian.so
+
+## Usage
+
+Run `./build.sh` to get the Triton Marian backend shared library.
+
+For all the users, you can put the libtriton_marian.so into the following places:
+
+- <model_repository>/<model_name>/<version_directory>/libtriton_marian.so
+- <model_repository>/<model_name>/libtriton_marian.so
+
+For the AzureML Inference team members, you can put it into the following place of *aml-triton* base image:
+
+- <backend_directory>/marian/libtriton_marian.so
+
+Where <backend_directory> is by default /opt/tritonserver/backends.
+
+## Make changes
+
+If you want to compile with another version of Marian, you need to replace `RUN git checkout youki/quantize-embedding` in the Dockerfile, then copy the new CMakeLists.txt replace the old one, add src/cmarian.cpp into CMakeLists.txt and make some changes to make sure it will build a static library of Marian.
```

Comment:
@delong-coder Does this work only with `youki/quantize-embedding`? Is there a reason why we cannot try to merge that branch into master? Maybe @ykim362 is in a better position to comment.

Reply:
> @delong-coder Does this work only with `youki/quantize-embedding`? Is there a reason why we cannot try to merge that branch into master? Maybe @ykim362 is in a better position to comment.

No, it can work with the master by doing a little change. We built this backend for the NLX-Sequence-Modeling of the Office team, they used `youki/quantize-embedding` branch, so we did the same thing. We will consider compiling this with the master of Marian.

Reply:
Great, thanks for the clarification.